serverless.yml

```diff
@@ -381,6 +381,7 @@ functions:
   index-apy-snapshots:
     handler: src/indexers/apy-snapshots-indexer.refreshApySnapshots
     timeout: 120
+    memorySize: 4096
```

Comment:
mem allocation increase here as we were hitting a limit on FD and this seems to have helepd